### PR TITLE
Remove underscore

### PR DESCRIPTION
**Pull request by roberttod**
- Didn't really need this dependancy

### CODE REVIEW

File: lib/local-storage-adapter.js [hunk 1]
```diff
@@ -130,20 +129,31 @@ function createLocalStorageAdapter (queueName) {
...
 
-  function now () {
-    return (new Date()).getTime()
+function onceOnDefer (fn) {
+  var called = false
+  return function debounced () {
+    if (called) return
+    called = true
+    setTimeout(function later () {
+      fn()
```
Comment (davidpadbury) [inline lib/local-storage-adapter.js]: shitty unhelpful suggestion: wouldn't `setTimeout(fn, 0)` work just as well?

File: lib/local-storage-adapter.js [hunk 2]
```diff
@@ -20,7 +19,7 @@ function createLocalStorageAdapter (queueName) {
   var setPending = false
   var queueCache = []
 
-  var flushOnDefer = _.debounce(flush, 0)
+  var flushOnDefer = onceOnDefer(flush)
```
Comment (davidpadbury) [inline lib/local-storage-adapter.js]: the `once`ing is the only real functional change. Isn't this a little significant as `flushOnDefer` gets called when updating the queue? Why the once?

Comment (roberttod) [inline lib/local-storage-adapter.js]: Should be
```js
function onceOnDefer (fn) {
  var called = false
  return function debounced () {
    if (called) return
    called = true
    setTimeout(function later () {
      fn()
      called = false
    }, 0)
  }
}
```
worrying that it wasn't caught, I will take a look at the tests.

Comment (roberttod) [inline lib/local-storage-adapter.js]: Okay, this should have the same functionality now and I added a test that caught the error.

Comment (davidpadbury) [inline lib/local-storage-adapter.js]: The old version didn't `once` (right?), why are we now?

Comment (roberttod) [inline lib/local-storage-adapter.js]: The old version would debounce, so if you call it multiple times it will only get called once. This would then reset after it is actually called. That is now the same that is happening here, I guess it is just way more explicit now.

Comment (davidpadbury) [inline lib/local-storage-adapter.js]: oohh. I was confused by the first implementation. You had the behavior of `_.once` wrapped in a `_.debounce` which gelled with the name `onceOnDefer`, so I was wondering why the change.

File: lib/local-storage-adapter.js (hunk 1 above)
Comment (roberttod) [inline lib/local-storage-adapter.js]: Are you suggesting this entire function could be replaced? I guess with setPending you could, the only difference is it stops us from deferring loads of flush calls that do nothing. Would be neater though, let me play with it.

**Comment by roberttod** [comment]
@rdecoene can we get release notes?

**Comment by rdecoene** [comment]
Release notes live

**Comment by rogy** [comment]
:+1:

File: lib/local-storage-adapter.js [hunk 3]
```diff
@@ -57,7 +54,7 @@ function createLocalStorageAdapter (queueName) {
     if (dirtyCache) {
       queueCache = json.parse(adapter.load(queueKey) || '[]')
       dirtyCache = false
-      flushOnDefer()
+      setTimeout(flush, 0)
```
Comment (KidkArolis) [inline lib/local-storage-adapter.js]: Hold up. Doesn't this behave differently. If you call debounce multiple times - it collapses those calls into 1. Whereas now, setTimeout will call each one individually. Given it's 0 timeout - probably not a big deal.

Comment (roberttod) [inline lib/local-storage-adapter.js]: Take a look at the comments above, @davidpadbury discussed this.
